### ISSUE TITLE
Site enhancements: red update

### ISSUE THREAD

**Issue by Jessdwilson** [ObjectiveSubject/cgu#282]
With the design changes can we please update the red from #b72029 to #af1e27 for buttons etc. as well as the text on the Why CGU stats? With the other color changes in the design updates we shouldn't encounter any accessibility issues since the new, darker red shouldn't appear over black in the navigation but will instead appear over a light grey which according to the WebAIM contrast checker it passes

**Comment by kpettinga** [comment]
@Jessdwilson so instances where there is red on black, it should remain the original red?

**Comment by Jessdwilson** [comment]
@kpettinga in the new designs we shouldn't encounter any instances where the red appears over black, but let me know if that's incorrect

**Comment by kpettinga** [comment]
updated on [cgudesign.wpengine.com](cgudesign.wpengine.com)
